x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/components/suggested_curations_callout.test.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import '../../../__mocks__/engine_logic.mock';
+
+import { setMockValues } from '../../../../__mocks__/kea_logic';
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+import { cloneDeep, set } from 'lodash';
+
+import { SuggestionsCallout } from '../../curations/components/suggestions_callout';
+
+import { SuggestedCurationsCallout } from './suggested_curations_callout';
+
+const MOCK_VALUES = {
+  // EngineLogic
+  engine: {
+    search_relevance_suggestions: {
+      curation: {
+        pending: 1,
+      },
+    },
+  },
+};
+
+describe('SuggestedCurationsCallout', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    setMockValues(MOCK_VALUES);
+  });
+
+  it('renders', () => {
+    const wrapper = shallow(<SuggestedCurationsCallout />);
+
+    expect(wrapper.is(SuggestionsCallout));
+  });
+
+  it('is empty when the suggestions are undefined', () => {
+    setMockValues({ ...MOCK_VALUES, engine: {} });
+
+    const wrapper = shallow(<SuggestedCurationsCallout />);
+
+    expect(wrapper.isEmptyRender()).toBe(true);
+  });
+
+  it('is empty when no pending curations', () => {
+    const values = cloneDeep(MOCK_VALUES); // set mutates so we cloneDeep first
+    set(values, 'engine.search_relevance_suggestions.curation.pending', 0);
```

Comment:
This is weird, but it seems more readable than `{...MOCK_VALUES, {...MOCK_VALUES.curation, curation: etc etc`, and I think its better than just writing a whole new object, because this highlights specifically whats changed vs the defaults.  But it is kinda weird (I wish `lodash.set` didn't mutate the source object)

Reply:
See comment above about lodash fp

Reply:
Thanks for the tip! I did this in a2a3a8a